Next 13 Module build failed UnhandledSchemeError

### Environment

self-hosted (https://develop.sentry.dev/self-hosted/)

### Steps to Reproduce

1. `npx create-next-app@latest`
```
√ What is your project named? 
√ Would you like to use TypeScript with this project? Yes
√ Would you like to use ESLint with this project?  Yes
√ Would you like to use Tailwind CSS with this project?  No
√ Would you like to use `src/` directory with this project?  Yes
√ Use App Router (recommended)?  Yes
√ Would you like to customize the default import alias? Yes
√ What import alias would you like configured? ... @/*
```

2. `npx @sentry/wizard@latest -i nextjs`
3.  `pnpm build` 


### Expected Result

Build should be successfull

### Actual Result

Failed to compile.
```
C:Users<name><path-to-project>sentry.server.config.ts
Module build failed: UnhandledSchemeError: Reading from "C:Users<name><path-to-project>sentry.server.config.ts" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "c:" URIs.
```

dependencies
```
    "@sentry/nextjs": "^7.52.1",
    "@types/node": "20.1.5",
    "@types/react": "18.2.6",
    "@types/react-dom": "18.2.4",
    "eslint": "8.40.0",
    "eslint-config-next": "13.4.2",
    "next": "13.4.2",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "typescript": "5.0.4"
```


### Product Area

Issues

### Link

_No response_

### DSN

_No response_

### Version

_No response_

Here is the output:
```ts
TEST LOG {                                                                                                    
  sentryConfigFilePath: 'C:\\Users\\tim\\Documents\\Repositories\\next-sentry-app\\sentry.server.config.ts',  
  rp: 'C:\\Users\\tim\\Documents\\Repositories\\next-sentry-app\\src\\app\\layout.tsx',                       
  imp: '..\\..\\..\\sentry.server.config.ts'                                                                  
}                                                                                                             
TEST LOG {                                                                                                    
  sentryConfigFilePath: 'C:\\Users\\tim\\Documents\\Repositories\\next-sentry-app\\sentry.server.config.ts',  
  rp: 'C:\\Users\\tim\\Documents\\Repositories\\next-sentry-app\\src\\app\\page.tsx',                         
  imp: '..\\..\\..\\sentry.server.config.ts'                                                                  
}                                                                                                             
```

After the error we see:
```
Import trace for requested module:
C:Users imDocumentsRepositoriesext-sentry-appsentry.server.config.ts
./src/app/page.tsx

./src/app/layout.tsx
Module not found: Can't resolve '......sentry.server.config.ts'

https://nextjs.org/docs/messages/module-not-found

./src/app/page.tsx
Module not found: Can't resolve '......sentry.server.config.ts'

https://nextjs.org/docs/messages/module-not-found
```

So it looks like there are two issues:
- We need to `.replace(/\\/g, '/') 
- We need to `path.dirname(resourcePath)`

Like:
```ts
templateCode = `import "${path.relative(path.dirname(this.resourcePath), sentryConfigFilePath).replace(/\\/g, '/')}";\n`.concat(templateCode);
``

--- a/packages/nextjs/src/config/loaders/wrappingLoader.ts
+++ b/packages/nextjs/src/config/loaders/wrappingLoader.ts
@@ -203,7 +203,15 @@ export default function wrappingLoader(
     }
 
     if (sentryConfigFilePath) {
-      templateCode = `import "${sentryConfigFilePath}";`.concat(templateCode);
+      let importPath = sentryConfigFilePath;
+
+      // absolute paths do not work with Windows
+      // https://github.com/getsentry/sentry-javascript/issues/8133
+      if (path.isAbsolute(importPath)) {
+        importPath = path.relative(path.dirname(this.resourcePath), importPath);
+      }
+
+      templateCode = `import "${importPath.replace(/\\/g, '/')}";\n`.concat(templateCode);
     }
   } else if (wrappingTargetKind === 'middleware') {
     templateCode = middlewareWrapperTemplateCode;

